src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -534,6 +534,29 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== Upgrade/Downgrade/Backup ProActive using Docker Compose
+
+The ProActive version could be easily upgraded using Docker Compose. To do so, we support automatic upgrade, downgrade, and backup of ProActive Server installations in Docker.
```

Comment:
I suggest `ProActive can be easily upgraded using Docker Compose.`

Reply:
ok